src/main/webapp/app/exercises/shared/structured-grading-criterion/grading-instructions-details/grading-instructions-details.component.ts

```diff
@@ -188,10 +192,30 @@ export class GradingInstructionsDetailsComponent implements OnInit, AfterContent
         return UsageCountCommand.identifier + ' ' + instruction.usageCount;
     }
 
+    initializeExerciseGradingInstructionText(): string {
+        if (this.exercise.gradingInstructions) {
+            return this.exercise.gradingInstructions + '\n' + '\n';
+        } else {
+            return 'Add Assessment Instruction text here' + '\n' + '\n';
```

Comment:
[nit] join two newlines
```suggestion
            return 'Add Assessment Instruction text here' + '\n\n';
```